<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodingOptionsExperementalWidget</class>
 <widget class="QWidget" name="EncodingOptionsExperementalWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>386</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>386</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Samples length of 
encoding frame:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Length of encoding 
superframe:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Maximum sample length 
of rang:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Minimum sample length 
of rang:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Relative shifting 
between domains:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Mode of mixing 
of stereo channels:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>2048 samples</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="superFrameLengthspinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>32 samples</string>
      </property>
     </widget>
     <widget class="QComboBox" name="minSampleLengthRangcomboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>32 samples</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16 samples</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8 samples</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 samples</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="realtiveDomainShiftspinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
     </widget>
     <widget class="QComboBox" name="mixingChannelModecomboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>321</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SIDE</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
